shotgun_api3/shotgun.py

```diff
@@ -2155,6 +2156,132 @@ def upload(self, entity_type, entity_id, path, field_name=None, display_name=Non
         is_thumbnail = (field_name in ["thumb_image", "filmstrip_thumb_image", "image",
                                        "filmstrip_image"])
 
+        # Thumbnails can only be uploaded to SG for now...
```

Comment:
might be clearer to say something like "Review quicktimes are handled as a special case and uploaded directly to amazon S3"